.changeset/dry-ghosts-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-studio': patch
+---
+
+Exporting V1_Runtime, V1_EngineRuntime and V1_engineRuntimeModelSchema
```

Comment:
This is a minor technical change, you don't need a changelog message. You can keep this file, but no need for any message. Thanks!